database/schema/V1.0__schema.sql

```diff
@@ -1,126 +1,1356 @@
+
+
+--
+-- Name: user_session user_session_id; Type: DEFAULT; Schema: application; Owner: postgres
+--
+
+ALTER TABLE ONLY application.user_session ALTER COLUMN user_session_id SET DEFAULT nextval('application.user_session_user_session_id_seq'::regclass);
+
+
+--
+-- Name: lookup lookup_id; Type: DEFAULT; Schema: reference; Owner: postgres
+--
+
+ALTER TABLE ONLY reference.lookup ALTER COLUMN lookup_id SET DEFAULT nextval('reference.lookup_lookup_id_seq'::regclass);
+
+
+--
+-- Name: lookup_type lookup_type_id; Type: DEFAULT; Schema: reference; Owner: postgres
+--
+
+ALTER TABLE ONLY reference.lookup_type ALTER COLUMN lookup_type_id SET DEFAULT nextval('reference.lookup_type_lookup_type_id_seq'::regclass);
+
+
+--
+-- Name: supplier_product_capability supplier_product_capability_id; Type: DEFAULT; Schema: reference; Owner: postgres
+--
+
+ALTER TABLE ONLY reference.supplier_product_capability ALTER COLUMN supplier_product_capability_id SET DEFAULT nextval('reference.supplier_product_capability_supplier_product_capability_id_seq'::regclass);
+
+
+--
+-- Data for Name: email_recipient; Type: TABLE DATA; Schema: application; Owner: postgres
+--
+
+COPY application.email_recipient (email_template_id, email_address) FROM stdin;
+\.
+
+
+--
+-- Data for Name: email_template; Type: TABLE DATA; Schema: application; Owner: postgres
+--
+
+COPY application.email_template (email_template_id, subject, body) FROM stdin;
+\.
+
+
+--
+-- Data for Name: site_attribute; Type: TABLE DATA; Schema: application; Owner: postgres
+--
+
+COPY application.site_attribute (site_attribute_id, site_definition_id, site_attribute_name, site_attribute_value, added_date, last_updated) FROM stdin;
+\.
+
+
+--
+-- Data for Name: site_definition; Type: TABLE DATA; Schema: application; Owner: postgres
+--
+
+COPY application.site_definition (site_definition_id, site_ods_code, site_party_key, site_asid, site_unique_identifier, added_date, last_updated, site_definition_status_id, site_interactions, master_site_unique_identifier) FROM stdin;
+\.
+
+
+--
+-- Data for Name: site_definition_status; Type: TABLE DATA; Schema: application; Owner: postgres
+--
+
+COPY application.site_definition_status (site_definition_status_id, site_definition_status_name) FROM stdin;
+\.
+
+
+--
+-- Data for Name: user; Type: TABLE DATA; Schema: application; Owner: postgres
+--
+
+COPY application."user" (user_id, email_address, last_logon_date, is_admin, added_date, authorised_date) FROM stdin;
+\.
+
+
+--
+-- Data for Name: user_session; Type: TABLE DATA; Schema: application; Owner: postgres
+--
+
+COPY application.user_session (user_session_id, user_id, start_time, end_time) FROM stdin;
+\.
+
+
+--
+-- Data for Name: lookup; Type: TABLE DATA; Schema: reference; Owner: postgres
+--
+
+COPY reference.lookup (lookup_id, lookup_value, lookup_type_id, added_date, disabled_date, linked_lookup_id) FROM stdin;
```

Comment:
Do we need to make these actual INSERT statements? Could the hardcoded dates be replaced with now() ?